uv.lock

```diff
@@ -2390,7 +2390,7 @@ wheels = [
 
 [[package]]
 name = "xdsl"
-version = "0.25.0+43.g10221499"
+version = "0.25.0+47.g46c79300"
```

Comment:
Unrelated, but does it make sense for xdsl itself to be in the lock file? Is this just how uv manages things?